buildSrc/src/main/kotlin/com.hedera.hashgraph.application.gradle.kts

```diff
@@ -0,0 +1,16 @@
+import org.gradle.kotlin.dsl.version
+
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+}
+
+repositories {
+}
+
+dependencies {
+}
+
```

Comment:
Rather than using these generated buildSrc files, I wonder if we can just use a simple build.gradle.kts at the top level instead.

Reply:
Or maybe we can reuse the files we use in other projects? I'm not sure how well we can reuse them. Is there a way to reuse conventions across repos?

Reply:
I forgot to remove this file. I was going to use it to group dependencies, but went the catalog route instead. I'll just remove the it.